src/main/kotlin/de/livepoll/api/util/AccountListener.kt

```diff
@@ -29,15 +32,34 @@ class AccountListener: ApplicationListener<OnCreateAccountEvent> {
         accountService.createVerificationToken(user, token)
 
         val recipientAddress = user.email
-        val subject = "Account Confirmation Live-Poll"
-        val confirmationUrl = event.appUrl + "/v1/account/confirm?token=$token"
-        val message = "Please confirm your email: "
-
-        javaMailSender.send(SimpleMailMessage().apply {
-            setTo(recipientAddress)
-            setSubject(subject)
-            setText("$message\r\n$serverUrl$confirmationUrl")
-        })
-    }
+        val subject = "Your Live-Poll registration"
+        val confirmationUrl = "$frontendUrl/activate/$token"
 
+        val mimeMessage: MimeMessage = javaMailSender.createMimeMessage()
+        val helper = MimeMessageHelper(mimeMessage, true)
+        helper.setFrom("noreply@live-poll.de")
+        helper.setTo(recipientAddress)
+        helper.setSubject(subject)
+        helper.setText(
+                "<html>"
+                        + "<body>"
+                        + "<img src='cid:logo' style='float:top;width:150px;height:150px;'/>"
```

Comment:
Did you send a test mail to check if the `src='cid:logo` is working properly?

Reply:
Email looks good. I tested it.

Reply:
In Mail for Windows, the image is a bit huge, but I think that is not that important now.